wafer/sponsors/templates/wafer.sponsors/sponsors.html

```diff
@@ -6,16 +6,21 @@
 <div class="wafer list">
     {% for sponsor in sponsor_list %}
     <section class="wafer wafer-sponsor">
-    <div>
-        <h2>
-           {% if sponsor.url %}
-           <a href="{{ sponsor.url }}">{{ sponsor.name }}</a>
-           {% else %}
-           {{ sponsor.name }}
-           {% endif %}
-        </h2>
-        {{ sponsor.description.rendered|safe }}
+    <div class="well">
+      <h2 class="bs-callout bs-callout-default bs-callout-{{ sponsor.packages.first.name.lower }}">
+         {% if sponsor.packages.first.name.lower == 'platinum' %}
```

Comment:
Adding a field to the model definitely sounds like the best solution.

On Tue, Aug 23, 2016 at 12:43 PM, Neil Muller <notifications@github.com>
wrote:

> In wafer/sponsors/templates/wafer.sponsors/sponsors.html
> <https://github.com/CTPUG/wafer/pull/277#discussion_r75841695>:
>
> > @@ -6,16 +6,21 @@
> >  <div class="wafer list">
> >      {% for sponsor in sponsor_list %}
> >      <section class="wafer wafer-sponsor">
> > -    <div>
> > -        <h2>
> > -           {% if sponsor.url %}
> > -           <a href="{{ sponsor.url }}">{{ sponsor.name }}</a>
> > -           {% else %}
> > -           {{ sponsor.name }}
> > -           {% endif %}
> > -        </h2>
> > -        {{ sponsor.description.rendered|safe }}
> > +    <div class="well">
> > +      <h2 class="bs-callout bs-callout-default bs-callout-{{ sponsor.packages.first.name.lower }}">
> > +         {% if sponsor.packages.first.name.lower == 'platinum' %}
>
> We've added a order attribute to sponsor packages, so we could potentially
> use that. It's still rather hacky, but easily controllable via the site
> admin.
>
> It might be worthwhile adding the character to display as a field to the
> package model, for more flexibility?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/CTPUG/wafer/pull/277/files/811426f2f1b1048c782c249a5895e9eb0f94d7ca#r75841695>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACdAzS7z4tPu6jm1rqF05ZSZsVXyaR_4ks5qis7CgaJpZM4JJvXL>
> .
>